README.md

```diff
@@ -7,6 +7,12 @@
 
 A Javascript module made to simplify communication with Sia Skynet portals from the browser.
 
+## Updating to v3 from v2
+
+The latest stable major version is `v3`. There are many breaking changes from `v2`.
+
+Please consult [the update guide](https://siasky.net/docs/v3-beta/?javascript--browser#updating-from-v2) for help migrating your code.
```

Comment:
I think you had me change this URL in the blog article -- will this be accurate after the docs update?